python/lsst/sdm/schemas/lsstcam.yaml

```diff
@@ -5567,6 +5567,17 @@ tables:
     description: Forced PSF flux uncertainty measured on the direct image.
     datatype: float
     ivoa:unit: nJy
+  - name: templateFlux
+    description: Forced photometry flux for a point source model measured on the template image
+      centered at the DiaObject position.
```

Comment:
I know you're just backporting existing wording, but I would expect this to be on the DIASource position.